handbook/business-operations/README.md

```diff
@@ -867,6 +867,7 @@ If a contract is ready for signature and requires no review or revision, the req
 When a contract is going to be routed for signature by someone outside of Fleet (i.e. the vendor or customer), the requestor is responsible for working with the other party to make sure the document gets routed to the CEO for signature.
 
 The SLA for contract signature is **2 business days**. Please do not follow up on signature unless this time has elapsed.
+> _Note: signature open time for the CEO is not currently measured, to avoid the overhead of creating separate signature issues to measure open and close time. This may change as signature volume increases._
```

Comment:
add newline above to avoid future copy/paste issues and/or any surprises with markdown compilation  (less risky / seems to work better with one newline of separation between formatted sections like this.  This is somewhat superstition on my part and likely informed by old broken implementations of this same website publishing system that has been evolving since @uncletammy built it and @rachaelshaw and @eashaw and I reworked it in the years since)

Reply:
For the sake of consistency, bold `**Note:**`

For example, from https://fleetdm.com/handbook/business-operations#hiring-a-new-team-member:
<img width="937" alt="image" src="https://github.com/fleetdm/fleet/assets/618009/2f5b9a98-7590-48b8-b671-2972e6aad89b">

Reply:
And capitalize the first word of the sentence: `Signature`